src/us/kbase/workspace/database/Util.java

```diff
@@ -20,15 +18,54 @@ public static void xorNameId(final String name, final Long id, final String type
 					type, name, id));
 		}
 	}
-	
+
 	/** Check if a string is null or whitespace only.
 	 * @param s the string to test.
 	 * @return true if the string is null or whitespace only, false otherwise.
 	 */
 	public static boolean isNullOrEmpty(final String s) {
 		return s == null || s.trim().isEmpty();
 	}
-	
+
+	/** Throws a null pointer exception if an object is null.
```

Comment:
No content changes here; I just moved all the methods that detect nulls / spaces to the same place in the file.